prisma/schema.prisma

```diff
@@ -10,24 +10,98 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Exhibitor {
-  id                 String  @id @default(uuid())
+model ExhibitorInterestRegistration {
```

Comment:
I think we should create a schema diagram for reference. Added a ticket to the board for that but feel free to mark it as done if you already have one.